tests/subsys/canbus/isotp/conformance/prj.conf

```diff
@@ -7,3 +7,5 @@ CONFIG_ISOTP_ENABLE_CONTEXT_BUFFERS=y
 CONFIG_ISOTP_RX_BUF_COUNT=6
 CONFIG_ISOTP_RX_BUF_SIZE=128
 CONFIG_ISOTP_RX_SF_FF_BUF_COUNT=2
+# These tests require millisecond tick resolution
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
```

Comment:
This will lead to problems with nordic MCUs, which run their sys clock on the RTC crystal with `CONFIG_SYS_CLOCK_TICKS_PER_SEC=32768`. Probably this setting should be changed only for `native_posix[64]` boards.

Reply:
I agree. Having to set this for a test suite is not optimal. However, it needs to be set for not only `native_posix` and `native_posix_64`, but also all non-tickless system timers as they default to 100Hz tick as well.

Reply:
@martinjaeger Instead of changing the tick frequency for this test suite I have opted to just skip the STmin test if `CONFIG_SYS_CLOCK_TICKS_PER_SEC` is too low. Please re-review.

Reply:
That's also a clean solution. Thanks.